x/feesplit/handler.go

```diff
@@ -19,13 +19,18 @@ func NewHandler(k keeper.Keeper) sdk.Handler {
 		ctx.SetEventManager(sdk.NewEventManager())
 
 		if !tmtypes.HigherThanVenus3(ctx.BlockHeight()) {
-			errMsg := fmt.Sprintf("feesplt module not supprt at height %d", ctx.BlockHeight())
+			errMsg := fmt.Sprintf("feesplt module not support at height %d", ctx.BlockHeight())
 			return nil, sdkerrors.Wrap(sdkerrors.ErrUnknownRequest, errMsg)
 		}
 
+		params := k.GetParams(ctx)
```

Comment:
Why this line can not use GetParamsWithCache

Reply:
GetParamsWithCache be used for evm hook